<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>561</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>O Controller</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1" colspan="9">
     <widget class="QtCharts::QChartView" name="graphicsView"/>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>26</pointsize>
       </font>
      </property>
      <property name="text">
       <string>I</string>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="2" column="9">
     <widget class="QPushButton" name="VNewButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>41</height>
       </size>
      </property>
      <property name="text">
       <string>设置目标角度</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>26</pointsize>
       </font>
      </property>
      <property name="text">
       <string>P</string>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QLabel" name="label_10">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>26</pointsize>
       </font>
      </property>
      <property name="text">
       <string>D</string>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>41</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QRadioButton" name="ManuRButton">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>10</y>
         <width>61</width>
         <height>19</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>手动</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="AutoRButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>61</width>
         <height>19</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>自动</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="PlineEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>41</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="ClearButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>41</height>
       </size>
      </property>
      <property name="text">
       <string>清空图像</string>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="QLineEdit" name="DlineEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>41</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLineEdit" name="IlineEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>41</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="CloseButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>41</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { background-color: rgb(255, 150, 150); border-radius: 3px; color: rgb(255, 255, 255); } QPushButton:hover { background-color: rgb(255, 100, 100); }</string>
      </property>
      <property name="text">
       <string>关闭</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5" colspan="3">
     <widget class="QPushButton" name="SetNewButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>41</height>
       </size>
      </property>
      <property name="text">
       <string>设置新参数</string>
      </property>
     </widget>
    </item>
    <item row="1" column="9">
     <widget class="QLineEdit" name="VlineEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>41</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="cursorPosition">
       <number>0</number>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="8">
     <widget class="QLabel" name="label_11">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>26</pointsize>
       </font>
      </property>
      <property name="text">
       <string>V</string>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="3">
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>460</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>有线连接</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>串口选择</string>
          </property>
          <property name="textFormat">
           <enum>Qt::MarkdownText</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="COMCBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>波特率</string>
          </property>
          <property name="textFormat">
           <enum>Qt::MarkdownText</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="BaudCBox">
          <item>
           <property name="text">
            <string>115200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>76800</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>57600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>38400</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>19200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>14400</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4800</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2400</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1200</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>停止位</string>
          </property>
          <property name="textFormat">
           <enum>Qt::MarkdownText</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="EndCBox">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>0</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1.5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>数据位</string>
          </property>
          <property name="textFormat">
           <enum>Qt::MarkdownText</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="DataCBox">
          <item>
           <property name="text">
            <string>8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>校验位</string>
          </property>
          <property name="textFormat">
           <enum>Qt::MarkdownText</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="CheckCBox">
          <item>
           <property name="text">
            <string>None</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Odd</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Even</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QPushButton" name="SearchSeiralButton">
          <property name="text">
           <string>搜索串口</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="QPushButton" name="ConnectSerialButton">
          <property name="text">
           <string>连接串口</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="2">
         <widget class="QPushButton" name="CloseSerialButton">
          <property name="text">
           <string>关闭串口</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>无线连接</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBoxIP">
          <property name="title">
           <string/>
          </property>
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>111</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>IP地址</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="IPEdit">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>20</y>
             <width>151</width>
             <height>41</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBoxPort">
          <property name="title">
           <string/>
          </property>
          <widget class="QLabel" name="label_7">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>111</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>端口号</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="PortEdit">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>20</y>
             <width>151</width>
             <height>41</height>
            </rect>
           </property>
           <property name="text">
            <string>1883</string>
           </property>
          </widget>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QGroupBox" name="groupBoxPortSub">
          <property name="title">
           <string/>
          </property>
          <widget class="QLabel" name="label_13">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>111</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>订阅主题（收）</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="SubEdit">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>20</y>
             <width>151</width>
             <height>41</height>
            </rect>
           </property>
           <property name="text">
            <string>Callback</string>
           </property>
          </widget>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QGroupBox" name="groupBoxPub">
          <property name="title">
           <string/>
          </property>
          <widget class="QLabel" name="label_15">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>111</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>发布主题（发）</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="PubEdit">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>20</y>
             <width>151</width>
             <height>41</height>
            </rect>
           </property>
           <property name="text">
            <string>Command</string>
           </property>
          </widget>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="WLConnectButton">
          <property name="text">
           <string>连接MQTT</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="WLDisconnectButton">
          <property name="text">
           <string>断开连接</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>848</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QGraphicsView</extends>
   <header>qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
